package.json

```diff
@@ -8,13 +8,15 @@
     "eject": "expo eject"
   },
   "dependencies": {
```

Comment:
Libraries related to react-navigation, react-native-screens and react-native-web can be let go for now.

Reply:
Yeah I'm fine with ripping out anything and everything at this point.

Reply:
Will remove them then.